packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -32,9 +33,10 @@ type SelectOpenerProps = AriaProps & {
      * id should match the field id.
      */
     id?: string;
-    //TODO: error state
-    // error: boolean,
-
+    /**
+     * Whether or not the input has an invalid value. Defaults to false.
+     */
+    isInvalid: boolean;
```

Comment:
**suggestion:** As I mentioned in the first comment, it would be better using the `error` name here to describe that state for consistency purposes across the Design System (e.g. `Checkbox`, `Radio`, `TextField` all use `error` to define this state).